--- previous
+++ current
@@ -1,4 +1,28 @@
 services:   radarr:     image: linuxserver/radarr:latest+    container_name: radarr-4k+    environment:+      - PUID=1000+      - PGID=1000+      - TZ=Europe/London+      - UMASK=022+    volumes:+      - /mnt/nvme/docker/radarr-4k:/config+      - /mnt/nas/data:/data+      - /mnt/nvme/scripts:/scripts +    ports:+      - 3021:7878/tcp+    networks:+      - ichor+    security_opt:+      - no-new-privileges:true+    restart: always+    labels:+      - "com.centurylinklabs.watchtower.enable=true"++networks:+  ichor:+    external: true+