--- previous
+++ current
@@ -1,4 +1,25 @@
+version: "3.9"+ services:   wizarr:     image: ghcr.io/wizarrrr/wizarr:latest+    container_name: wizarr+    restart: unless-stopped +    volumes:+      - /mnt/nvme/docker/wizarr:/data++    environment:+      - TZ=Europe/London+      - WIZARR_JELLYFIN_URL=https://jellyfin.mortalsrv.uk+      - WIZARR_JELLYFIN_API_KEY=14ca69c0a8724b4c9bbcd45660295b8b++    ports:+      - "8081:5690"++    networks:+      - ichor++networks:+  ichor:+    external: true